Tests/AlicerceTests/Logging/Loggers/LoggerTestCase.swift

```diff
@@ -101,6 +101,42 @@ class LoggerTestCase: XCTestCase {
 
         log.log(level: .verbose, message: "message", file: "filename.ext", line: 1337, function: "function")
     }
+
+    func testLog_WithLogDestinationAndLogLevelMeetingMinLevel_ShouldNotInvokeWrite() {
+
+        let minLevel = Log.Level.verbose
+        let level = Log.Level.error
+
+        XCTAssert(level.meets(minLevel: minLevel))
+
+        let log = MockLogDestination()
+        log.mockMinLevel = minLevel
+
+        log.writeInvokedClosure = { item, _ in
+            XCTAssertEqual(item.level, level)
+            XCTAssertEqual(item.message, "message")
+            XCTAssertEqual(item.file.description, "filename.ext")
+            XCTAssertEqual(item.line, 1337)
+            XCTAssertEqual(item.function.description, "function")
+        }
```

Comment:
shouldn't it be the same as the following test if we're expecting it to not invoke the write?

```suggestion
         log.writeInvokedClosure = { _, _ in XCTFail("Unexpected write call") }
```

Reply:
You are correct, forgot to correctly rename the functio 😓

Reply:
happens to us all 😌